client/src/app/financial-organizer/FinancialOrganizer.js

```diff
@@ -1,64 +1,249 @@
+//     </div>
+//   );
+// };
+
+// export default function FinancialOrganizer() {
+//   const [expenses, setExpenses] = useState([]);
+//   const type = "transaction";
+
+//   const fetchExpenses = async () => {
+//     try {
+//       const res = await axios.get(`http://localhost:3001/get/${type}`);
+//       setExpenses(res.data);
+//     } catch (error) {
+//       console.error("Error fetching data: ", error);
+//     }
+//   };
+
+//   useEffect(() => {
+//     fetchExpenses();
+//   }, []);
+
+//   const handleDelete = async (id) => {
+//     try {
+//       await axios.delete(`http://localhost:3001/delete/${type}/${id}`, {
+//         withCredentials: true,
+//       });
+//       setExpenses(expenses.filter((expense) => expense.expense_id !== id));
+//     } catch (error) {
+//       console.error("Error deleting expense:", error);
+//     }
+//   };
+
+//   return (
+//     <>
+//       <NavBar />
+//       <div className="ps-5 ms-4 me-4 mt-4">
+//         <Row>
+//           {/* Left side of the page */}
+//           <Col md={12} lg={4} className="mb-4">
+//             {/* Summary overview */}
+//             <FinancialOverview />
+
+//             {/* TODO: create an in-file component for "Summary" */}
+//             <div className="summary">
+//               <h3>Your Summary</h3>
+//               <div className="summary-details">
+//                 <div>
+//                   <p>You've spent</p>
+//                   <p>SGD 300</p>
+//                 </div>
+//                 <div>
+//                   <p>Budget left</p>
+//                   <p>SGD 1200</p>
+//                 </div>
+//               </div>
+//             </div>
+//           </Col>
+
+//           {/* Right side of the page */}
+//           <Col md={12} lg={8}>
+//             <div className="transactions-container">
+//               {/* Form component */}
+//               <Form type="transaction" fetchData={fetchExpenses} />
+
+//               {/* Entry-output component; type = transaction */}
+//               <div className="transaction-list">
+//                 {expenses.length > 0 ? (
+//                   expenses.map((expense) => (
+//                     <EntryOutput
+//                       type="transaction"
+//                       key={expense.expense_id}
+//                       id={expense.expense_id}
+//                       date={expense.created_at}
+//                       title={expense.expense_name}
+//                       category={expense.expense_category}
+//                       description={expense.expense_description}
+//                       amount={expense.expense_amount}
+//                       onDelete={handleDelete}
+//                     />
+//                   ))
+//                 ) : (
+//                   <p>No expenses to display.</p>
+//                 )}
+//               </div>
+//             </div>
+//           </Col>
+//         </Row>
+//       </div>
+//     </>
+//   );
+// }
```

Comment:
nitpick: If these comments wont be of use, can delete them and then commit right after.

By doing so, it is easier to revert the deletion in case you actl need the code :3 thankew